go.mod

```diff
@@ -348,3 +354,5 @@ replace (
 replace github.com/hashicorp/memberlist => github.com/grafana/memberlist v0.3.1-0.20220708130638-bd88e10a3d91
 
 replace golang.org/x/net => golang.org/x/net v0.17.0
+
+replace go.opentelemetry.io/collector/receiver/otlpreceiver => github.com/jpkrohling/opentelemetry-collector/receiver/otlpreceiver v0.0.0-20240320225130-6e76aa23e8d3
```

Comment:
I didn't see the change mentioned merged into this fork, but perhaps I missed it.

Reply:
It is actually vendoring a branch from this fork https://github.com/jpkrohling/opentelemetry-collector/commits/otlpreciever-http-response-code/ that has the commit in it.

Reply:
It got merged so I'm vendoring the new changes.